src/miniscript/types/extra_props.rs

```diff
@@ -119,6 +119,14 @@ pub struct ExtData {
     pub max_dissat_size: Option<(usize, usize)>,
     /// The timelock info about heightlocks and timelocks
     pub timelock_info: TimeLockInfo,
+    /// Maximum stack + alt stack size during satisfaction execution
+    /// This does **not** include initial witness elements. This element only captures
+    /// the additional elements that are pushed during execution.
+    pub exec_stack_elem_count_sat: Option<usize>,
```

Comment:
We have very distinct naming across methods and fields: some places use `stackelem`, some `stack_elem`. According to Rust API guidelines, `stack_elem` is more preferable

Reply:
Yes, I will do this as a separate PR.